cli/smartcontract/testdata/rpcbindings/notifications/notifications.go

```diff
@@ -24,6 +28,13 @@ func Array() {
 	runtime.Notify("SomeArray", [][]int{})
 }
 
+// Update updates the contract with a new one.
+func Update(script, manifest []byte, data any) {
+	ctx := storage.GetReadOnlyContext()
+	mgmt := storage.Get(ctx, mgmtKey).(interop.Hash160)
```

Comment:
It won't work. Contract storage doesn't contain storage item with `mgmtKey` value, someone have to put it there.

Reply:
Revert changes of `notifications.go` contract. We have a test for `nft-nd`, it's enough.